wdl-lint/tests/lints/unknown_rule/source.wdl

```diff
@@ -1,6 +1,13 @@
+#@ except: SnackCase
```

Comment:
```suggestion
#@ except: SnackCase, ThisIsNotARealRule, MissingMetas
```

We want to test both paths for the rule (and suppress an unrelated warning from the test).

Reply:
@a-frantz - Any objection to continuing `MissingMetas` here or should this get updated to include minimal meta blocks?

Reply:
I'm fine leaving `MissingMetas` excepted

Reply:
This needs to be applied.